src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/ClassDefHandler.java

```diff
@@ -68,19 +68,24 @@ protected DetailAST getListChild() {
     @Override
     public void checkIndentation() {
         final DetailAST modifiers = getMainAst().findFirstToken(TokenTypes.MODIFIERS);
+        final DetailAST ident = getMainAst().findFirstToken(TokenTypes.IDENT);
+        if (getMainAst().getType() == TokenTypes.ANNOTATION_DEF) {
+            checkWrappingIndentation(getMainAst(), modifiers);
```

Comment:
I removed this and got no differences in test. Please explain why it is needed and have a example in input to show it.